### PR TITLE
Require to use at least desktop client 2.0 by default

### PR DESCRIPTION
**Pull request by MorrisJobke**
- downstreams owncloud/core#26441

cc @LukasReschke @nickvergessen @rullzer @karlitschek

### CODE REVIEW

File: apps/dav/lib/Connector/Sabre/BlockLegacyClientPlugin.php [hunk 1]
```diff
@@ -68,7 +68,7 @@ public function beforeHandler(RequestInterface $request) {
 			return;
 		}
 
-		$minimumSupportedDesktopVersion = $this->config->getSystemValue('minimum.supported.desktop.version', '1.7.0');
+		$minimumSupportedDesktopVersion = $this->config->getSystemValue('minimum.supported.desktop.version', '2.0.0');
```
Comment (nickvergessen) [inline apps/dav/lib/Connector/Sabre/BlockLegacyClientPlugin.php]: Btw sure we can allow people to require more up to date versions.
But should we have 2.0.0 as a minimum as well, so when someone copied sample.config.php he now still requires 2.0.0?

**Comment by karlitschek** [comment]
👍

**Comment by LukasReschke** [comment]
LGTM

File: apps/dav/lib/Connector/Sabre/BlockLegacyClientPlugin.php (hunk 1 above)
Comment (rullzer) [inline apps/dav/lib/Connector/Sabre/BlockLegacyClientPlugin.php]: As long as we have no pacakges for all platforms lets just keep this as is :)

**Pull request merged**